src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportSteps.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import {
+    Steps,
+} from "antd";
+
+export function SampleMetadataImportSteps({currentStep}) {
+    const { Step } = Steps;
```

Comment:
Move this out of the function as it would make this happen on each render.  Put it below the imports